docs/USAGE.md

```diff
@@ -39,6 +39,20 @@ try {
 }
 ```
 
+### Fetching one editorial article by id for publication id
```

Comment:
add explanation for `editorial` - it is not so intuitive. Maybe sth like `editorial (with status different than published)`.